cmake/templates/oneDALConfig.cmake.in

```diff
@@ -29,6 +29,8 @@ endif()
 # ONEDAL_USE_DPCPP: yes | no. Default is yes
 # ONEDAL_INTERFACE: yes | no. Default is no
 # ONEDAL_SET_TBB_MANUALLY: yes | no. Default is no
+# MKL_DEPENDENCY: yes | no. Default is no
```

Comment:
Why MKL_DEPENDENCY is mentioned here if it's internal variable, not handle argument?

Reply:
good catch, will be fixed